x/simulation/simulate.go

```diff
@@ -130,7 +130,7 @@ func SimulateFromSeed(
 		pastVoteInfos,
 		eventStats.Tally,
 		1,
-		genesisTimestamp,
+		blockTime,
```

Comment:
Line 109, blockTime = genesisTimeStamp.
Is this then doing anything?

Reply:
not really, I'll change it to be blockTime everywhere